applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -87,6 +87,20 @@ <h3 class="signup-confirmation--success">Check your Inbox!</h3>
                     </div>
                 </a>
             </div>
+            @if(NewslettersRemoveConfirmationStep.isSwitchedOn) {
+                <div class="newsletters-heading__divider"></div>
+                <div class="newsletters-privacy-notice__container">
+                    <div class="newsletters-privacy-notice__heading">
+                        Privacy Notice
+                    </div>
+                    <div class="newsletters-privacy-notice__content">
+                        <p>We thought you should know this newsletter may also contain information about Guardian products,
+                        services and chosen charities or online advertisements.</p>
+                        <p>Newsletters mays also contain content funded by outside parties.</p>
+                        <p>You can read more about our privacy policy  <a data-link-name="privacy-policy" href="/help/privacy-policy">here</a>.</p>
```

Comment:
Its slightly better for accessibility to include 'our privacy policy' in the text of the link - if the text is just 'here' it can be confusing on screen readers
```suggestion
                        <p>You can read more about <a data-link-name="privacy-policy" href="/help/privacy-policy">our privacy policy here</a>.</p>
```